<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemChargeDialog</class>
 <widget class="QDialog" name="MemChargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MemChargeDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBarWidget" name="titleWnd" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2 {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>17</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="refreshButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#refreshButton{
	color:#ffffff;
	font-size:16px;
	border-image: url(:/shareImages/sure_normal.png);
}
                        
QPushButton#refreshButton:pressed{
	border-image: url(:/shareImages/sure_press.png);
}</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cancelButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>66</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#cancelButton{
	color:#ffffff;
	font-size:16px;
	border-image: url(:/shareImages/cancel_normal.png);
}
                        
QPushButton#cancelButton:pressed{
	border-image: url(:/shareImages/cancel_press.png);
}</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>328</width>
          <height>93</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="rechargeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>21</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:14px;</string>
         </property>
         <property name="text">
          <string>充值金额：</string>
         </property>
        </widget>
        <widget class="QLabel" name="statusLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>56</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:14px;</string>
         </property>
         <property name="text">
          <string>支付状态：</string>
         </property>
        </widget>
        <widget class="QLabel" name="rechargeAmtLabel">
         <property name="geometry">
          <rect>
           <x>216</x>
           <y>21</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font:bold;
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>￥37.02</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="statusValueLabel">
         <property name="geometry">
          <rect>
           <x>246</x>
           <y>56</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:14px;
color: rgb(117, 171, 87);</string>
         </property>
         <property name="text">
          <string>成功</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
